share/src/main/java/io/zensoft/share/model/diesel/DefaultHeadersValues.java

```diff
@@ -0,0 +1,20 @@
+package io.zensoft.share.model.diesel;
+
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+public class DefaultHeadersValues {
```

Comment:
how you feel about if you move all classes related to `headers` into `headers` package
i think there would be a lot less of mess